site/docs/contribute/modules.md

```diff
@@ -23,13 +23,13 @@ Modules that access or write data need credentials in order to access the data s
 An example for Vault Login API call which uses the Vault parameters is as follows:
 
 ```
-$ curl -v --request POST <address>/<authPath> -H "Content-Type: application/json" --data '{"jwt": <module service account token>, "role": <role>}'
+$ curl -v -X POST <address>/<authPath> -H "Content-Type: application/json" --data '{"jwt": <module service account token>, "role": <role>}'
```

Comment:
@eres Would this work with the latest kind release?

Reply:
@shlomitk1 yes. if you refer to the error reported on https://github.com/mesh-for-data/mesh-for-data/issues/672 so the fix is when configuring Vault (not when retrieving  the secrets), more sepcifcally the following `issuer` should be added to the vault configuration:
`vault write auth/kubernetes/config issuer="https://kubernetes.default.svc.cluster.local" kubernetes_host="https://kubernetes.default.svc:443"`